### PR TITLE
fix: [Bug-4420] [Landing Page] Added accessbility changes

### PR DESCRIPTION
**Pull request by Rohini-Microsoft**
**Bug**: https://dev.azure.com/CSACTOSOL/CSA%20Solutioning/_workitems/edit/4420

**1. Automated check: Label**
Issue: Form elements must have labels (label - https://accessibilityinsights.io/info-examples/web/label)

**2. Automated check: button-name**
Issue: Buttons must have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)

 **3. Images**
Open BYOAI URL.
Open inspect.
Alternative text for Images.
Expected result: Need to add an alternative text for image.

Screenshot 2024-07-03 160700

Screenshot 2024-07-03 160611

### CODE REVIEW

**Comment by Roopan-Microsoft** [comment]
@Rohini-Microsoft - I see the Icon and background got impacted due to the change. Please check those. Added the screen recording for comparision.

https://github.com/microsoft/Build-your-own-AI-Assistant-Solution-Accelerator/assets/168007406/cf2239f1-f97a-4974-94ed-9d9169243364

File: App/frontend/src/components/Homepage/Cards.tsx [hunk 1]
```diff
@@ -120,14 +120,14 @@ export const TextFieldCard = ({ className, ...props }: CardProps) => {
           textAlign: "start",
         }}
       >
-        <Title3 
+       <h2 
           style={{
             fontSize: "1.2rem",
-            marginBottom: "2px",
+            fontWeight: "500"
```
Comment (Roopan-Microsoft) [inline App/frontend/src/components/Homepage/Cards.tsx:126]: @Rohini-Microsoft - Please check the impacted places as mentioned in the comments

**Review by Roopan-Microsoft — CHANGES_REQUESTED** [review]
Please check the impacted places as mentioned in the comments

File: App/frontend/src/components/Homepage/Cards.tsx (hunk 1 above)
Comment (Rohini-Microsoft) [reply App/frontend/src/components/Homepage/Cards.tsx]: Resolved

**Review by Roopan-Microsoft — APPROVED** [review]
Approving the changes

**Pull request merged**